homework/hackyourtemperature/app.js

```diff
@@ -0,0 +1,41 @@
+import express from "express";
+import fetch from "node-fetch";
+import { API_KEY } from "./sources/keys.js";
+
+const app = express();
+let weatherInfo = {};
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("hello from backend to frontend");
+});
+
+app.post("/weather", async (req, res) => {
+  const cityName = req.body.cityName;
```

Comment:
You can also use object destructuring here since the value you are retrieving and the variable name are the same.

```suggestion
  const { cityName } = req.body;
```

Reply:
Updated. Thanks for the feedback 🙂